tests/store/test_version_filters.py

```diff
@@ -0,0 +1,57 @@
+from housekeeper.store.api.core import Store
+from housekeeper.store.models import Version
+from housekeeper.store.version_filters import filter_version_by_id
+
+
+def test_filter_version_by_id_returns_the_correct_version(populated_store: Store):
+    """Test getting version by version ID."""
+
+    # GIVEN a store with a version
+    version: Version = populated_store.session.query(Version).first()
```

Comment:
```suggestion
    version: Version = populated_store._get_version_query.first()
```
Let not couple query to the test suit